From bf990a5e81f2af5c33583cc500eb655ad1bf173d Mon Sep 17 00:00:00 2001
From: "RALOVICH, Kristof" <tade60@freemail.hu>
Date: Thu, 27 Mar 2014 09:42:48 +0100
Subject: [PATCH 1/2] try fixing FTBFS under hurd-i386, kfreebsd-amd64,
 kfreebsd-i386
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

                from /«PKGBUILDDIR»/src/common.cpp:20:
/«PKGBUILDDIR»/src/Log.hpp: In constructor 'antpm::Log::Log(const char*)':
/«PKGBUILDDIR»/src/Log.hpp:176:8: error: '::access' has not been declared
     if(::access(logFileName, 0x00) != -1)
        ^
make[4]: *** [CMakeFiles/antpm.dir/common.cpp.o] Error 1
make[4]: *** Waiting for unfinished jobs....
make[4]: Leaving directory `/«PKGBUILDDIR»/cmake-build'
make[3]: *** [CMakeFiles/antpm.dir/all] Error 2
make[3]: Leaving directory `/«PKGBUILDDIR»/cmake-build'
make[2]: *** [all] Error 2
make[2]: Leaving directory `/«PKGBUILDDIR»/cmake-build'
make[1]: *** [override_dh_auto_build] Error 2
make[1]: Leaving directory `/«PKGBUILDDIR»'
make: *** [build-arch] Error 2
---
 src/Log.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/Log.hpp
+++ b/src/Log.hpp
@@ -29,7 +29,7 @@
 # include <crtdbg.h>
 # include <io.h>
 #endif
-#ifdef __linux__
+#if defined(__linux__) || defined(__GNU__) || defined(__FreeBSD__) || defined(__FreeBSD_kernel__)
 # include <unistd.h>
 #endif
 #include <iostream>
